data/build.gradle

```diff
@@ -31,14 +31,24 @@ android {
     }
 }
 
+preBuild.dependsOn ktlint
+
 dependencies {
 
     implementation project(path: ':domain')
 
-    implementation 'androidx.core:core-ktx:1.6.0'
-    implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation 'com.google.android.material:material:1.4.0'
-    testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    def lib = rootProject.ext.generalDependencies
+
+    //basics
+    implementation lib.coreKtx
+    implementation lib.appCompat
+
+    //UI
+    implementation lib.material
+    implementation lib.constraintLayout
+
+    //test
+    testImplementation lib.junit
+    androidTestImplementation lib.extJunit
+    androidTestImplementation lib.espresso
```

Comment:
are you sure you need these dependencies here?

Reply:
Removed test dependencies